### ISSUE TITLE
Don't automatically generate PDFs (for some documents)

### ISSUE THREAD

**Issue by emilyyyylime** [nvarner/typst-lsp#402]
- I have searched the [issues](https://github.com/nvarner/typst-lsp/issues) of this repo and believe that this is not a duplicate.
- I have searched the [discussions](https://github.com/nvarner/typst-lsp/discussions) and believe that my question is not already covered.

## Feature Request
Sometime, especially when working on libraries/templates/packages, it is undesirable to generate a `file.pdf` alongside your `file.typ`. Some way to confgure the default behaviour for new projects and/or per-project config, or somehow detecting when a file generates an empty pdf (and only exports symbols) would be really useful.

**Comment by ngoetti** [comment]
Should be somewhat solved with this: https://github.com/nvarner/typst-lsp/pull/389

**Comment by emilyyyylime** [comment]
I don't think this will work for my setup. For one sometimes I work on packages in isolation (at least within the context of the editor) so I need some way to completely disable pdf generation for the current session. Secondly, Helix isn't quite hackable enough to run any action like that automatically yet.

**Comment by Ciflire** [comment]
Rn, I just use helix with zellij (multiplexer) with a layout that only compiles the main.typ
